src/main/java/duke/TaskList.java

```diff
@@ -0,0 +1,147 @@
+package duke;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+/**
+ * Represents a list of Task items.
+ */
+public class TaskList {
+    private Task[] tasks;
+    private int cmdNum = 0;
+    TaskList(Task[] t, int  n){
+        tasks = t;
+        cmdNum = n;
+    }
+
+    /**
+     * Adds new task to list.
+     *
+     * @param t Task to be added to the list.
+     * */
```

Comment:
Perhaps you can add ```@return``` to provide some description on what this method is trying to return (noticed the same point throughout the code)

Reply:
Yep, forgot to update some of the JavaDocs after editing the code from Week 3. Thanks for the catch!